api/filter_dimensions.go

```diff
@@ -27,19 +26,20 @@ func (api *FilterAPI) getFilterBlueprintDimensionsHandler(w http.ResponseWriter,
 	vars := mux.Vars(r)
 	filterBlueprintID := vars["filter_blueprint_id"]
 	logData := log.Data{"filter_blueprint_id": filterBlueprintID}
```

Comment:
So in the dataset API I added an `action` variable to logData as I believe it is useful when debugging environments and understanding what went wrong with that service. Would it be worth doing the same in the filter API?

For example here:
`logData := log.Data{"filter_blueprint_id": filterBlueprintID, "action": getDimensionsAction}`